rules/defaults.yaml

```diff
@@ -389,9 +397,70 @@
 		RequiresCondition: firedamage
 	ExternalCondition@firedamage:
 		Condition: firedamage
+	DamagedByTerrain@firedamage1:
```

Comment:
Use SelfHeal instead on an external template to decrease duplication. This is ludicrous. SelfHeal even has damagetype support to trigger flamedeath if the actor is killed by this even.